From 501a65311e421576aca58484d85039afc49a5a81 Mon Sep 17 00:00:00 2001
From: Weiwen Chen <cww@rock-chips.com>
Date: Mon, 10 Aug 2020 19:22:02 +0800
Subject: [PATCH] ARM: configs: rv1126-spi-nor.config: add leds pwm driver

Update by:
    make ARCH=arm rv1126_defconfig
    cp .config rv1126.config
    make ARCH=arm rv1126_defconfig rv1126-spi-nor.config
    make ARCH=arm menuconfig
    scripts/diffconfig -m rv1126.config .config > arch/arm/configs/rv1126-spi-nor.config

Signed-off-by: Weiwen Chen <cww@rock-chips.com>
Change-Id: I7b3b087ef62094eebb90a22ef2fde8ce150f340b
---
 arch/arm/configs/rv1126-spi-nor.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rv1126-spi-nor.config b/arch/arm/configs/rv1126-spi-nor.config
index da77bbd945ca..174c3d3637db 100644
--- a/arch/arm/configs/rv1126-spi-nor.config
+++ b/arch/arm/configs/rv1126-spi-nor.config
@@ -18,6 +18,7 @@ CONFIG_JFFS2_FS=y
 CONFIG_KERNEL_XZ=y
 CONFIG_KEYBOARD_ADC=m
 CONFIG_KEYBOARD_GPIO=m
+CONFIG_LEDS_PWM=m
 # CONFIG_LEDS_TRIGGERS is not set
 CONFIG_MAC80211=m
 CONFIG_MMC_DW=m
-- 
2.35.3

